analyzer/src/main/kotlin/managers/utils/PythonInspector.kt

```diff
@@ -57,13 +57,17 @@ internal object PythonInspector : CommandLineTool {
         workingDir: File,
         definitionFile: File,
         pythonVersion: String = "38",
+        operatingSystem: String = "linux"
```

Comment:
Shouldn't this use `OPTION_OPERATING_SYSTEM_DEFAULT`?

Reply:
How do we maintain this / recognize if the defaults of the python-inspector tool change?

Reply:
Manually, I guess. But maybe I remove those defaults altogether in a later iteration when refactoring the tests as mentioned in some other comment, because we might not need default values here anymore. In any way `PythonInspector` is a tool used by `Pip` so I wouldn't like it to directly depend on configuration options of `Pip`.